src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy

```diff
@@ -201,7 +201,8 @@ class DevTask extends AbstractFeatureTask {
     void setContainerfile(String containerfile) {
         if (containerfile != null) {
             // ensures the containerfile is defined with the full path - matches how maven behaves
-            this.containerfile = convertParameterToCanonicalFile(containerfile, "containerfile");      
+            this.containerfile = convertParameterToCanonicalFile(containerfile, "containerfile");
+            project.liberty.dev.containerfile = this.containerfile;
```

Comment:
Actually, I guess that is the point of this change...when the setDockerfile method calls this method to set the containerfile, it will set it in the project itself. Then that automatically gets used by dev mode. Right?

Reply:
Yup, then this gets set in the new runner because the DevExtension value is set here. That is handled in `processContainerParams()` at 1349.

Reply:
This behavior was initially added for the `container` parameter but not the rest of the properties.